Add option to skip the song in working memory

rbong opened the pull request:
Added the option `--skip-working` to skip the song in working memory when the project is unspecified.

Let me know if you'd like the argument to be called something else. `--skip-wm` seemed too vague and `--skip-working-memory` seemed too long. I didn't add a short form because it seems hard to capture the meaning in a single character (`-s` also seems too vague).

The program fails when both `--working-memory` is given and `--skip-working` because it feels like that behaviour should explicitly be undefined. `-i -1` does not fail because `--name` and `--index` both cause the option to be ignored.

Related to #65

stijnfrishert commented:
First of all: thanks for wanting to work on liblsdj! Your help is very much appreciated :)

Your code looks perfectly fine to merge, though there's one small problem: I use the `master` branch for always up-to-date releases, and develop further on `develop`.

Would it be possible for you to merge `develop` into your branch, fix any merge conflicts, and request to merge this back in `develop`? I've started using a different third-party library for command-line argument parsing (to ditch Boost), so you encounter some small differences there.

rbong commented:
@stijnfrishert No problem, I've rebased the `develop` branch and fixed conflicts to keep the commit history clean and updated the target of this merge request. That should be better

stijnfrishert commented:
Great! :D
